paimon-core/src/main/java/org/apache/paimon/disk/SpillableBuffer.java

```diff
@@ -53,14 +53,18 @@ public class ExternalBuffer {
 
     // The size of each segment
     private final int segmentSize;
+    private final boolean spillable;
 
     private final List<ChannelWithMeta> spilledChannelIDs;
     private int numRows;
 
     private boolean addCompleted;
 
-    public ExternalBuffer(
-            IOManager ioManager, MemorySegmentPool pool, AbstractRowDataSerializer<?> serializer) {
+    public SpillableBuffer(
```

Comment:
Don't change this name... The renaming is not so solid.

Reply:
reverted